airflow/jobs/triggerer_job.py

```diff
@@ -382,8 +382,8 @@ def update_triggers(self, requested_trigger_ids: Set[int]):
         # handling.
         current_trigger_ids = set(self.triggers.keys())
         # Work out the two difference sets
-        new_trigger_ids = requested_trigger_ids.difference(current_trigger_ids)
-        old_trigger_ids = current_trigger_ids.difference(requested_trigger_ids)
+        new_trigger_ids = requested_trigger_ids - current_trigger_ids
+        remove_trigger_ids = current_trigger_ids - requested_trigger_ids
```

Comment:
```suggestion
        trigger_ids_to_remove = current_trigger_ids - requested_trigger_ids
```

nit -- no strong preference

Reply:
@dstandish Do you want to change this? (Personally I like the `...to_remove` name slightly better as well since `remove_...` sounds like a verb, not a noun.)

Reply:
yeah i do agree with the suggestion... but this superceded by other PRs and i will close

thank you for helping to review